### ISSUE TITLE
Confirm and add contact details to accessibility statement

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/human-services#113]

**Comment by MikeThacker1** [comment]
@UsmanAfzal I think these should be Porism details for now. I'd prefer not to make public a phone number. Ask me for what you need.

**Comment by UsmanAfzal** [comment]
@MikeThacker1 i'll structure the content as below with the contact details only /contact-us ideally with a form to submit. I don't think this page has been completed as of yet design/content. I have also referenced Porism as the maintainers of the website

### Feedback and contact information
[Tell us if you need information in a different format.](https://openreferraluk.org/contact-us)
In your message, include:
- the web address (URL) of the content
- your email address and name
- the format you need - for example, plain text, braille, BSL, large print or audio CD

### Reporting accessibility problems with this website
If you find any problems that are not listed on this page or you think we’re not meeting the accessibility requirements, [contact us](https://openreferraluk.org/contact-us).

**Comment by MikeThacker1** [comment]
@UsmanAfzal if you don't need a named individual, are you able to use hello@openreferraluk.org rather than my own email address?

**Comment by UsmanAfzal** [comment]
Yes and is preferable to use a form on the contact us page rather than the email visible but let's see.
